src/js/core/services/ui-grid-util.js

```diff
@@ -886,8 +886,8 @@ module.service('gridUtil', ['$log', '$window', '$document', '$http', '$templateC
       };
       this._purgeQueue();
       if (aSync){ //Do this asynchronysly
-        var promise = $timeout(focusBySelector);
-        this.queue.push($timeout(focusBySelector));
+        var promise = $timeout(focusBySelector, 0, false);
+        this.queue.push($timeout(focusBySelector), 0, false);
```

Comment:
@csvan there is a mistake here, should be:

```js
this.queue.push($timeout(focusBySelector, 0, false));
```

right?

Reply:
Yes that's a mistake